docs/SPRITES.md

```
@@ -0,0 +1,476 @@
+| `kimi-code` | `kimi-code` | Moonshot Kimi K2.5 |
+| `claude` | `claude` | Anthropic Claude Code |
+
+Configure via `--agent` flag or `BB_AGENT` environment variable.
+
+---
+
+## Provisioning Workflow
+
+When you run `bb dispatch`, BB executes this workflow:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)